goms-infrastructure/src/main/kotlin/com/goms/v2/persistence/auth/repository/RefreshTokenRepositoryImpl.kt

```diff
@@ -0,0 +1,18 @@
+package com.goms.v2.persistence.auth.repository
+
+import com.goms.v2.domain.auth.RefreshToken
+import com.goms.v2.persistence.auth.mapper.RefreshTokenMapper
+import com.goms.v2.repository.auth.RefreshTokenRepository
+import org.springframework.stereotype.Component
+
+@Component
+class RefreshTokenRepositoryImpl(
+    private val refreshTokenJpaRepository: RefreshTokenJpaRepository,
+    private val refreshTokenMapper: RefreshTokenMapper
+) : RefreshTokenRepository{
+    override fun save(refreshToken: RefreshToken) {
+        val refreshTokenEntity = refreshTokenMapper.toEntity(refreshToken)
+            refreshTokenJpaRepository.save(refreshTokenEntity)
+        }
```

Comment:
코드가 많이 들여쓰기가 안되있는거 같습니다.
```suggestion
override fun save(refreshToken: RefreshToken) {
        val refreshTokenEntity = refreshTokenMapper.toEntity(refreshToken)
        refreshTokenJpaRepository.save(refreshTokenEntity)
}
```

Reply:
eef67e06a603e5ddd8f6c885f788d4e8a7f550d4 수정했습니다.